a10_octavia/controller/worker/tasks/service_group_tasks.py

```diff
@@ -56,6 +58,15 @@ def execute(self, pool, vthunder):
         axapi_client = self.client_factory(vthunder)
```

Comment:
Per review comment of set method. Let's instead add a try catch here which handles exceptions coming off the set method. We'll log and then raise.

The last operation of the execute method (that occurs within the try block) will be logging a success

Reply:
The update method should be altered to match this as well. Though it'll simply catch the exception and log

Reply:
Did this change.